server/mongodb/actions/User.ts

```diff
@@ -19,6 +19,16 @@ export async function getNonprofitUser(userId: Types.ObjectId) {
   return user;
 }
 
+export async function getChapterUsers(chapterId: Types.ObjectId) {
+  await dbConnect();
+
+  const users = await UserModel.find({ chapter: chapterId }).populate(
```

Comment:
Let's just select the properties we need here instead of returning entire user documents.  So the _id, name, and email, which also means no need to populate the chapter.

Reply:
I removed the populate but I kept the rest as is, just in case in the future we need to get chapter users for some other purpose that requires more properties.

Reply:
Sounds good